.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Same here. Please remove the `.vscode` subfolder and maybe add it to `gitignore`.